src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -31,14 +46,34 @@ const isAzureRuntimeWrapper = (obj: any): obj is AzureRuntimeWrapper => {
   return castObj && !!castObj.workspaceId && !!castObj.runtimeName;
 };
 
+const getNormalizedComputeConfig = (config: RawRuntimeConfig): RuntimeConfig => ({
+  ...config,
+  normalizedRegion: getNormalizedComputeRegion(config),
+});
+
+const getNormalizedComputeRegion = (config: RawRuntimeConfig): NormalizedComputeRegion => {
```

Comment:
note that this is now a private method, and methods dealing with the raw config are not exposed in favor of only exposing the sanitized object.